ecs-init/docker/docker.go

```diff
@@ -84,6 +84,17 @@ const (
 	// pluginSpecFilesUsrDir specifies one of the locations of spec or json files
 	// of Docker plugins
 	pluginSpecFilesUsrDir = "/usr/lib/docker/plugins"
+	// iptableExecutableDir specifies the location of the iptable
```

Comment:
can you please %s/iptable/iptables/ everywhere?

Reply:
updated :)